domain-specific-datasets/app/defaults.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+SEED_DATA_PATH = "seed_data.json"
+PIPELINE_PATH = "pipeline.yaml"
+REMOTE_CODE_PATHS = ["defaults.py", "domain.py", "pipeline.py"]
+
+N_PERSPECTIVES = 5
+N_TOPICS = 5
+N_EXAMPLES = 5
+
+with open(SEED_DATA_PATH) as f:
+    DEFAULT_DATA = json.load(f)
+DEFAULT_DOMAIN = DEFAULT_DATA["domain"]
```

Comment:
do we want to use default data if people are supposed to clone and change this?

Reply:
The idea would not be to clone the seeding space. Instead, they would create a dataset from a single space (probably hosted on the DIBT org).

Personally, I find that the default data help to explain: What a domain, perspective, topic, and system prompt are.